cmd/clusterctl/client/cluster/crd_migration.go

```diff
@@ -230,3 +231,20 @@ func storageVersionForCRD(crd *apiextensionsv1.CustomResourceDefinition) (string
 	}
 	return "", errors.Errorf("could not find storage version for CRD %q", crd.Name)
 }
+
+// newCRDMigrationBackoff creates a new API Machinery backoff parameter set suitable for use with crd migration operations.
+// Clusterctl upgrades cert-manager right before doing CRD migration. This may lead to rollout of new certificates.
+// The time between new certificate creation + injection into objects (CRD, Webhooks) and the new secrets getting propagated
+// to the controller can be 60-90s, because the kubelet only periodically syncs secret contents to pods.
+// During this timespan conversion, validating- or mutationwebhooks may be unavailable and cause a failure.
+func newCRDMigrationBackoff() wait.Backoff {
+	// Return a exponential backoff configuration which returns durations for a total time of ~1m30s.
```

Comment:
Not opposed to it but the next line seems to sum up to a lot more than 90s?

Reply:
You should not sum it up, it are the points in time when the retry would happen.

Reply:
But ack, I will take a look that we increase some more, didn't think about the jitter, which maybe could make it to less than 90s!

Reply:
You are right, it does not :D